faq/faq.rst

```diff
@@ -59,13 +52,29 @@ Other queries which, by contrast, combine data from multiple nodes, do not suppo
 * Set operations
 * Transactional semantics for queries that span across multiple shards
 
+.. _faq_choose_shard_count:
+
 How do I choose the shard count when I hash-partition my data?
 --------------------------------------------------------------
-.. _faq_choose_shard_count:
 
-Optimal shard count is related to the total number of cores on the workers. Citus partitions an incoming query into its fragment queries which run on individual worker shards. Hence, the degree of parallelism for each query is governed by the number of shards the query hits. To ensure maximum parallelism, you should create enough shards on each node such that there is at least one shard per CPU core.
+One of the choices when first distributing a table is its shard count. This setting can be set differently for each table, and the optimal value depends on use-case. It is possible, but difficult, to change the count after cluster creation, so use these guidelines to choose the right size.
+
+In the :ref:`mt_blurb` use-case we recommend choosing between 32 - 128 shards.  For smaller workloads say <100GB, you could start with 32 shards and for larger workloads you could choose 64 or 128. This means that you have the leeway to scale from 32 to 128 worker machines.
+
+In the :ref:`rt_blurb` use-case, shard count should be related to the total number of cores on the workers. To ensure maximum parallelism, you should create enough shards on each node such that there is at least one shard per CPU core. We typically recommend creating a high number of initial shards, e.g. 2x or 4x the number of current CPU cores. This allows for future scaling if you add more workers and CPU cores.
+
+To choose a shard count for a table you wish to distribute, update the :code:`citus.shard_count` variable. This affects subsequent calls to :ref:`create_distributed_table`. For example
+
+.. code-block:: postgres
+
+  SET citus.shard_count = 64;
+  -- any tables distributed at this point will have
+  -- sixty-four shards
+
+How do I change the shard count for a hash partitioned table?
+-------------------------------------------------------------
 
-We typically recommend creating a high number of initial shards, e.g. 2x or 4x the number of current CPU cores. This allows for future scaling if you add more workers and CPU cores.
+Note that it is not straightforward to change the shard count of an already distributed table. If you need to do so, please `Contact Us <https://www.citusdata.com/about/contact_us>`_. It's good to think about shard count carefully at distribution time, see :ref:`faq_choose_shard_count`.
```

Comment:
I'm good to retain this note, or link to the other FAQ item, but I'd prefer to keep the other FAQ as well.

Reply:
It is actually the same question, but moved lower. Are you saying you prefer the old answer to the question over this new text?